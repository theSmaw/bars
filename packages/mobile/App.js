import React, { Component } from 'react';
import styled from 'styled-components';

export default class App extends Component {
  render() {
    return (
      <Container>
        <Welcome>
          Hello world
        </Welcome>
      </Container>
    );
  }
}

const Welcome = styled.Text`
  fontSize: 20px;
  textAlign: center;
  margin: 10px;
`;

const Container = styled.View`
  flex: 1;
  justifyContent: center;
  alignItems: center;
  backgroundColor: #004499
`;
