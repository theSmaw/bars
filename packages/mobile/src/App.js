import React from 'react';
import styled from 'styled-components/native';

export default function App() {
  return (
    <Container>
      <Welcome>
        Hello world
      </Welcome>
    </Container>
  );
}

const Welcome = styled.Text`
  font-family: Quicksand, sans-serif;
  font-size: 20px;
  text-align: center;
  margin: 10px;
`;

const Container = styled.View`
  flex: 1;
  justify-content: center;
  align-items: center;
  background-color: #004499
`;
