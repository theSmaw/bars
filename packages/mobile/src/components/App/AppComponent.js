import React from 'react';
import styled from 'styled-components/native';

import BarsComponent from '../Bars/BarsComponent'; // eslint-disable-line import/no-named-as-default

export default function AppComponent() {
  return (
    <Container>
      <BarsComponent />
    </Container>
  );
}

const Container = styled.View`
  flex: 1;
  justify-content: center;
  align-items: center;
  background-color: #fbf7f2
`;
