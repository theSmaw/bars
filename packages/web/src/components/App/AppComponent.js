import React from 'react';

import BarsComponent from '../Bars/BarsComponent'; // eslint-disable-line import/no-named-as-default

export default function AppComponent() {
  return (
    <div className="app">
      <BarsComponent />
    </div>
  );
}
