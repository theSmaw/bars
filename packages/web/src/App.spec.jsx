import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {

  it('Should render', () => {
    const app = shallow(<App />);

    expect(app).toMatchSnapshot();
  });
});
