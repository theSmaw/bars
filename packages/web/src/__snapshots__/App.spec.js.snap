// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render 1`] = `
<div>
  Hello World
</div>
`;
